### ISSUE TITLE
Add in fails to load assembly

### ISSUE THREAD

**Issue by shtirlitsDva** [chuongmep/CadAddinManager#9]
**Describe the bug**
When loading a dll the addin throws:

```
Loading AeciRibbonUiApi...Mono.Cecil.AssemblyResolutionException: Failed to resolve assembly: 'accoremgd, Version=24.2.0.0, Culture=neutral, PublicKeyToken=null'
   at Mono.Cecil.BaseAssemblyResolver.Resolve(AssemblyNameReference name, ReaderParameters parameters)
   at Mono.Cecil.DefaultAssemblyResolver.Resolve(AssemblyNameReference name)
   at Mono.Cecil.MetadataResolver.Resolve(TypeReference type)
   at Mono.Cecil.TypeReference.Resolve()
   at Mono.Cecil.Mixin.CheckedResolve(TypeReference self)
   at Mono.Cecil.SignatureReader.ReadCustomAttributeEnum(TypeReference enum_type)
   at Mono.Cecil.SignatureReader.ReadCustomAttributeElement(TypeReference type)
   at Mono.Cecil.SignatureReader.ReadCustomAttributeConstructorArguments(CustomAttribute attribute, Collection`1 parameters)
   at Mono.Cecil.MetadataReader.ReadCustomAttributeSignature(CustomAttribute attribute)
   at Mono.Cecil.CustomAttribute.<Resolve>b__35_0(CustomAttribute attribute, MetadataReader reader)
   at Mono.Cecil.ModuleDefinition.Read[TItem](TItem item, Action`2 read)
   at Mono.Cecil.CustomAttribute.Resolve()
   at Mono.Cecil.CustomAttribute.get_ConstructorArguments()
   at CadAddinManager.Model.AssemLoader.ResolveDuplicateMethod(String originalFilePath)
   at CadAddinManager.Model.AssemLoader.LoadAddinsToTempFolder(String originalFilePath, Boolean parsingOnly)
   at CadAddinManager.ViewModel.AddinManager.LoadAddin(String filePath, AssemLoader assemLoader) <Osnap off>
```

**Additional context**
The reference is declared in .csproj:

```
<Reference Include="accoremgd">
    <HintPath>C:\Program Files\Autodesk\AutoCAD 2023\accoremgd.dll</HintPath>
    <Private>False</Private>
</Reference>
```

**Comment by chuongmep** [comment]
Hi @shtirlitsDva , do you have any project sample to can help me create again problem ?

**Comment by chuongmep** [comment]
This issues can't create again, so I will close it like resolved.

**Comment by chuongmep** [comment]
@shtirlitsDva can give me to confirm this in your computer : 
The first problem I see is NuGet package use an update version higher version current in computer for this version, I also have same problem with that when I use NuGet package for reference, example : 
- Assembly accoremgd.dll version 2022 in my computer :  24.1.154.0
- Assembly accoremgd.dll version 2022 of NuGet package : 24.1.162.0
Nuget :
![image](https://user-images.githubusercontent.com/31106432/231906381-21fe7a63-4c43-4229-b391-a9d2a58b1818.png)
 My Computer : 
![image](https://user-images.githubusercontent.com/31106432/231906828-02b6a0bf-fe0f-4181-a48c-8ea3e5fb459b.png)

**Comment by shtirlitsDva** [comment]
Hi, thank you for your efforts. I don't remember what I had problems with the time I posted this bug report. I tried to use add-in manager again today, and now I get another error when I try to run a command:
Cannot load assembly. Error details: Autodesk.AutoCAD.Runtime.Exception: eDuplicateKey
   at Autodesk.AutoCAD.Runtime.CommandClass.AddCommand(ICommandLineCallable ca, MethodInfo mi)
   at Autodesk.AutoCAD.ApplicationServices.AutoCADApplicationHolder.Initialize(Assembly assembly)

Here's a project that doesn't work. Specifically command that is named: Intersect.massimporttdcyousee().

[IntersectUtilities.zip](https://github.com/chuongmep/CadAddinManager/files/11229469/IntersectUtilities.zip)

Also, I have many commands in this assembly and the list of commands does not list them fully when autocad is restarted and I open the add-in manager again. So, open add-in manager, load the dll, close civil, open civil, open add-in manager -> the list of commands is not complete, because there are so many of them.
